composer.json

```diff
@@ -50,7 +50,7 @@
     "oat-sa/extension-tao-lti": "15.5.0",
     "oat-sa/extension-tao-ltideliveryprovider": "12.9.0",
     "oat-sa/extension-tao-revision": "10.2.0",
-    "oat-sa/extension-tao-mediamanager": "12.20.3",
+    "oat-sa/extension-tao-mediamanager": "12.21.1",
```

Comment:
Why 12.20.3 to 12.21.1? should be 12.20.3 to 12.20.4 ?

Reply:
12.20.3 and 12.20.3 have only one difference: https://github.com/oat-sa/extension-tao-mediamanager/pull/359/files
all is about tests and does not affect business logic, so for me it does not make sense to cherry-pick and release one more minor version.
wdyt?

Reply:
I will update this PR